### ISSUE TITLE
Crash on cameramodel add

### ISSUE THREAD

**Issue by djjudas21** [camerahub/camerahub#660]
**Describe the bug**
```
2020-12-08T15:37:42.107419754Z stderr F Internal Server Error: /cameramodel/create/
2020-12-08T15:37:42.10745444Z stderr F Traceback (most recent call last):
2020-12-08T15:37:42.107465736Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
2020-12-08T15:37:42.107479035Z stderr F     response = get_response(request)
2020-12-08T15:37:42.107485191Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 145, in _get_response
2020-12-08T15:37:42.107491812Z stderr F     response = self.process_exception_by_middleware(e, request)
2020-12-08T15:37:42.10749865Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 143, in _get_response
2020-12-08T15:37:42.107504832Z stderr F     response = response.render()
2020-12-08T15:37:42.107510293Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/response.py", line 106, in render
2020-12-08T15:37:42.107516082Z stderr F     self.content = self.rendered_content
2020-12-08T15:37:42.107521865Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/response.py", line 83, in rendered_content
2020-12-08T15:37:42.107528638Z stderr F     content = template.render(context, self._request)
2020-12-08T15:37:42.107535532Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/backends/django.py", line 61, in render
2020-12-08T15:37:42.107561825Z stderr F     return self.template.render(context)
2020-12-08T15:37:42.107569149Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 171, in render
2020-12-08T15:37:42.107575138Z stderr F     return self._render(context)
2020-12-08T15:37:42.107581262Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 163, in _render
2020-12-08T15:37:42.107586401Z stderr F     return self.nodelist.render(context)
2020-12-08T15:37:42.107591624Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 937, in render
2020-12-08T15:37:42.107596919Z stderr F     bit = node.render_annotated(context)
2020-12-08T15:37:42.107601851Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 904, in render_annotated
2020-12-08T15:37:42.107606965Z stderr F     return self.render(context)
2020-12-08T15:37:42.107612028Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
2020-12-08T15:37:42.107617505Z stderr F     return compiled_parent._render(context)
2020-12-08T15:37:42.107623268Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 163, in _render
2020-12-08T15:37:42.107628692Z stderr F     return self.nodelist.render(context)
2020-12-08T15:37:42.107634004Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 937, in render
2020-12-08T15:37:42.107641609Z stderr F     bit = node.render_annotated(context)
2020-12-08T15:37:42.107647421Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 904, in render_annotated
2020-12-08T15:37:42.107653427Z stderr F     return self.render(context)
2020-12-08T15:37:42.107659522Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 62, in render
2020-12-08T15:37:42.107665462Z stderr F     result = block.nodelist.render(context)
2020-12-08T15:37:42.107671011Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 937, in render
2020-12-08T15:37:42.107676269Z stderr F     bit = node.render_annotated(context)
2020-12-08T15:37:42.107681535Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 904, in render_annotated
2020-12-08T15:37:42.107686539Z stderr F     return self.render(context)
2020-12-08T15:37:42.107693093Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/templatetags/crispy_forms_tags.py", line 203, in render
2020-12-08T15:37:42.107698448Z stderr F     c = self.get_render(context).flatten()
2020-12-08T15:37:42.107703912Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/templatetags/crispy_forms_tags.py", line 119, in get_render
2020-12-08T15:37:42.107709373Z stderr F     actual_form.form_html = helper.render_layout(
2020-12-08T15:37:42.107714678Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/helper.py", line 301, in render_layout
2020-12-08T15:37:42.10771961Z stderr F     html = self.layout.render(form, self.form_style, context, template_pack=template_pack)
2020-12-08T15:37:42.107728069Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/layout.py", line 135, in render
2020-12-08T15:37:42.107733494Z stderr F     return self.get_rendered_fields(form, form_style, context, template_pack, **kwargs)
2020-12-08T15:37:42.107740245Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/layout.py", line 96, in get_rendered_fields
2020-12-08T15:37:42.107745783Z stderr F     return "".join(
2020-12-08T15:37:42.107751485Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/layout.py", line 97, in <genexpr>
2020-12-08T15:37:42.107759595Z stderr F     render_field(field, form, form_style, context, template_pack=template_pack, **kwargs)
2020-12-08T15:37:42.107765231Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/utils.py", line 66, in render_field
2020-12-08T15:37:42.110212309Z stderr F     return field.render(form, form_style, context, template_pack=template_pack,)
2020-12-08T15:37:42.110234378Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/layout.py", line 289, in render
2020-12-08T15:37:42.110242157Z stderr F     fields = self.get_rendered_fields(form, form_style, context, template_pack, **kwargs)
2020-12-08T15:37:42.11024842Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/layout.py", line 96, in get_rendered_fields
2020-12-08T15:37:42.110254337Z stderr F     return "".join(
2020-12-08T15:37:42.110265116Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/layout.py", line 97, in <genexpr>
2020-12-08T15:37:42.110292357Z stderr F     render_field(field, form, form_style, context, template_pack=template_pack, **kwargs)
2020-12-08T15:37:42.110300108Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/utils.py", line 139, in render_field
2020-12-08T15:37:42.11030605Z stderr F     html = template.render(context)
2020-12-08T15:37:42.110311722Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/backends/django.py", line 61, in render
2020-12-08T15:37:42.110317021Z stderr F     return self.template.render(context)
2020-12-08T15:37:42.110322092Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 171, in render
2020-12-08T15:37:42.110327687Z stderr F     return self._render(context)
2020-12-08T15:37:42.110333602Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 163, in _render
2020-12-08T15:37:42.11033907Z stderr F     return self.nodelist.render(context)
2020-12-08T15:37:42.110344633Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 937, in render
2020-12-08T15:37:42.110350474Z stderr F     bit = node.render_annotated(context)
2020-12-08T15:37:42.110356711Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 904, in render_annotated
2020-12-08T15:37:42.11036272Z stderr F     return self.render(context)
2020-12-08T15:37:42.110368685Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 309, in render
2020-12-08T15:37:42.11037457Z stderr F     return nodelist.render(context)
2020-12-08T15:37:42.110379807Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 937, in render
2020-12-08T15:37:42.110384837Z stderr F     bit = node.render_annotated(context)
2020-12-08T15:37:42.110389995Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 904, in render_annotated
2020-12-08T15:37:42.110395363Z stderr F     return self.render(context)
2020-12-08T15:37:42.110400767Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 309, in render
2020-12-08T15:37:42.110405948Z stderr F     return nodelist.render(context)
2020-12-08T15:37:42.110410978Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 937, in render
2020-12-08T15:37:42.110416555Z stderr F     bit = node.render_annotated(context)
2020-12-08T15:37:42.110421746Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 904, in render_annotated
2020-12-08T15:37:42.1104272Z stderr F     return self.render(context)
2020-12-08T15:37:42.110433113Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 309, in render
2020-12-08T15:37:42.110438871Z stderr F     return nodelist.render(context)
2020-12-08T15:37:42.110444243Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 937, in render
2020-12-08T15:37:42.110449441Z stderr F     bit = node.render_annotated(context)
2020-12-08T15:37:42.110454802Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 904, in render_annotated
2020-12-08T15:37:42.110460382Z stderr F     return self.render(context)
2020-12-08T15:37:42.110473396Z stderr F   File "/usr/local/lib/python3.8/site-packages/crispy_forms/templatetags/crispy_forms_field.py", line 155, in render
2020-12-08T15:37:42.11047959Z stderr F     return str(field)
2020-12-08T15:37:42.110486965Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/utils/html.py", line 388, in <lambda>
2020-12-08T15:37:42.110492817Z stderr F     klass.__str__ = lambda self: mark_safe(klass_str(self))
2020-12-08T15:37:42.11049855Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/forms/boundfield.py", line 33, in __str__
2020-12-08T15:37:42.110545433Z stderr F     return self.as_widget()
2020-12-08T15:37:42.11063727Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/forms/boundfield.py", line 89, in as_widget
2020-12-08T15:37:42.110645375Z stderr F     return widget.render(
2020-12-08T15:37:42.110651286Z stderr F   File "/usr/local/lib/python3.8/site-packages/dal/widgets.py", line 151, in render
2020-12-08T15:37:42.110657243Z stderr F     widget = super(WidgetMixin, self).render(name, value, attrs, **kwargs)
2020-12-08T15:37:42.110664945Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/forms/widgets.py", line 241, in render
2020-12-08T15:37:42.110670511Z stderr F     context = self.get_context(name, value, attrs)
2020-12-08T15:37:42.110676175Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/forms/widgets.py", line 678, in get_context
2020-12-08T15:37:42.11068185Z stderr F     context = super().get_context(name, value, attrs)
2020-12-08T15:37:42.110691643Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/forms/widgets.py", line 638, in get_context
2020-12-08T15:37:42.11069712Z stderr F     context = super().get_context(name, value, attrs)
2020-12-08T15:37:42.110702702Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/forms/widgets.py", line 234, in get_context
2020-12-08T15:37:42.1107081Z stderr F     'attrs': self.build_attrs(self.attrs, attrs),
2020-12-08T15:37:42.11071387Z stderr F   File "/usr/local/lib/python3.8/site-packages/dal_select2_taggit/widgets.py", line 15, in build_attrs
2020-12-08T15:37:42.110719129Z stderr F     attrs = super(TaggitSelect2, self).build_attrs(*args, **kwargs)
2020-12-08T15:37:42.110725812Z stderr F   File "/usr/local/lib/python3.8/site-packages/dal_select2/widgets.py", line 140, in build_attrs
2020-12-08T15:37:42.110731282Z stderr F     attrs = super(TagSelect2, self).build_attrs(*args, **kwargs)
2020-12-08T15:37:42.110736738Z stderr F   File "/usr/local/lib/python3.8/site-packages/dal/widgets.py", line 60, in build_attrs
2020-12-08T15:37:42.110742296Z stderr F     if self.url is not None:
2020-12-08T15:37:42.110747869Z stderr F   File "/usr/local/lib/python3.8/site-packages/dal/widgets.py", line 166, in _get_url
2020-12-08T15:37:42.11075347Z stderr F     return reverse(self._url)
2020-12-08T15:37:42.11075869Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/urls/base.py", line 90, in reverse
2020-12-08T15:37:42.110780724Z stderr F     return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
2020-12-08T15:37:42.110792617Z stderr F   File "/usr/local/lib/python3.8/site-packages/django/urls/resolvers.py", line 673, in _reverse_with_prefix
2020-12-08T15:37:42.110798936Z stderr F     raise NoReverseMatch(msg)
2020-12-08T15:37:42.110804592Z stderr F django.urls.exceptions.NoReverseMatch: Reverse for 'tag-autocomplete' not found. 'tag-autocomplete' is not a valid view function or pattern name.
```

**Version of CameraHub**
0.24.2

**Comment by djjudas21** [comment]
Also occurs on lensmodel add

**Comment by djjudas21** [comment]
This was already fixed in c9b227c8
